src/components/SelectInput/index.ts

```diff
@@ -648,7 +648,7 @@ class SelectInput extends Element implements IBindable, IFocusable {
         }
 
         if (containerDom.firstChild) {
-            this._highlightLabel(containerDom.firstChild.ui);
+            this._highlightLabel(containerDom.firstChild.ui as Label);
```

Comment:
I'm assuming these elements can't be defined as a Label in the DOM already, so we don't need to cast?

Reply:
The `ui` property is explicitly typed as `Element` so we have to cast here because the `_highlightLabel` function demands a parameter of type `Label`. This isn't a runtime type check. This is a compile-time type check. So the TypeScript compiler doesn't actually know it's a `Label`.